woocommerce/google-listings-and-ads
# Get started page UX improvements: add get started card

❗ **Change the base branch to `feature/1392-ux-improvements-on-get-started-page` when #1449 is merged.**

### Changes proposed in this Pull Request:

Part of #1392, this PR adds the get started card to the new get started page.

Design for desktop: eQ9O4m2flzrcAiDMXkOW0m-fi-7350%3A121253
Design for mobile: eQ9O4m2flzrcAiDMXkOW0m-fi-7476%3A123161

### Screenshots:

#### Desktop

The image should appear at the right-hand side.

<img width="1344" alt="Screenshot 2022-04-29 at 14 47 23" src="https://user-images.githubusercontent.com/914406/165896922-5d65df15-3cb0-42da-a95b-d0c8821044a8.png">

#### Mobile

The image should appear at the top.

<img width="777" alt="Screenshot 2022-04-29 at 14 47 44" src="https://user-images.githubusercontent.com/914406/165896938-8c0d8818-795a-41ad-b524-2f1367b74137.png">

#### Tablet

The image should appear at the right-hand side.

<img width="834" alt="Screenshot 2022-04-29 at 14 48 02" src="https://user-images.githubusercontent.com/914406/165896809-ea92d7ad-4875-4d50-a646-436c5d3217fa.png">

### Detailed test instructions:

1. Go to GLA's get started page: `https://domain.test/wp-admin/admin.php?page=wc-admin&path=%2Fgoogle%2Fstart`
2. Check the fifth card of the get started page, the UI should align with the design.

#### Testing for Tracking Events

1. Open your console and run `localStorage.setItem('debug', 'wc-admin*')`.
2. Enable verbose mode in dev tool's console.
3. Go to GLA's get started page: `https://domain.test/wp-admin/admin.php?page=wc-admin&path=%2Fgoogle%2Fstart`
4. Click on the `Start listing products →` on the fifth card, it should open a new tab to the set up MC page.
    - `https://domain.test/wp-admin/admin.php?page=wc-admin&path=%2Fgoogle%2Fsetup-mc`
6. From the dev console you should see an event is sent with `target` = `set_up_free_listings`.

### Changelog entry

> Update - Add the get started card for UX improvements on get started page.
